phpstan.neon.dist

```diff
@@ -79,6 +79,11 @@ parameters:
             paths:
                 - lib/Doctrine/DBAL/Driver/PDOSqlsrv/Connection.php
 
+        -
+            message: '~Method Doctrine\\DBAL\\Driver\\PDOConnection\:\:lastInsertId\(\) should return string but returns string\|false\.~'
+            paths:
+                - lib/Doctrine/DBAL/Driver/PDOConnection.php
+
```

Comment:
@morozov This error seems to be raised because of the stub you contributed in /phpstan/phpstan-src#700.

The `mysqli_stmt::$affected_rows` property is documented as `int|string` which is also what the documentation on php.net says. The documentation does not say in which cases we can expect a string here, but my guess is that this happens if the number of affected rows is larger than `PHP_INT_MAX`.

However, this also applies to `mysqli_stmt::$num_rows` which your stub documents as `0|positive-int`.